visualization/app/codeCharta/ui/legendPanel/legendPanel.component.scss

```diff
@@ -6,17 +6,29 @@ legend-panel-component {
 		bottom: ($fixedHeight + 20) + px;
 		right: -35px;
 		transform: rotate(90deg);
-		z-index: 2;
+		border-radius: 0 0 5px 5px;
+
+		&.expanded {
```

Comment:
Is this expanded for the Attribute side panel, or the legendPanel?

Reply:
this moves the legendPanel to the left when the attributeSideBar is opened. same as with the viewCube. Should we rename the CSS class?

Reply:
Yes, I would use the same name as used for the viewCube CSS class

Reply:
Done